process.go

```diff
@@ -1,329 +1,92 @@
 package venom
 
 import (
-	"context"
-	"fmt"
-	"io/ioutil"
-	"os"
-	"path/filepath"
 	"strings"
 	"sync"
-	"time"
 
-	log "github.com/Sirupsen/logrus"
-	"github.com/fsamin/go-dump"
 	"gopkg.in/cheggaaa/pb.v1"
-	"gopkg.in/yaml.v2"
 )
 
-var aliases map[string]string
-var bars map[string]*pb.ProgressBar
-var mutex = &sync.Mutex{}
+var (
```

Comment:
That's better !

I know it's boring and i'm a bit annoying but may you share all of those vars by function parameters, and avoid global vars ?

example: `readFiles` is using `pb`and `chanToRun`, it's a bit confusing without reading two other files.